docs/installation/installation.rst

```diff
@@ -15,6 +12,12 @@ There are several ways to install Augur, ordered from least to most sophisticate
 
 .. tabs::
 
+   .. group-tab:: Nextstrain
+
+      Augur is part of the Nextstrain project, and is available in all Nextstrain runtimes.
+
+      Continue by following the :doc:`Nextstrain installation guide <docs.nextstrain.org:install>`.
```

Comment:
Let's add a bit after this saying how to access `augur` once you've installed `nextstrain`?

Reply:
Added in [force-push](https://github.com/nextstrain/augur/compare/46cef8bae70e548d6f6af5f9f4683e28bb407a8f..719ca1858ce7abf79a1762e2159627589f003217):

```
Once installed, you can use :doc:`cli:commands/shell` to run ``augur`` directly.
```